colorBarVGA/README.md

```diff
@@ -4,6 +4,18 @@ Simple VGA (640x480@60Hz) demonstration using a [Digilent PMOD
 VGA](https://digilent.com/reference/pmod/pmodvga/start) to displays 8 colors
 bars
 
+
+To use the design with PMOD-VGA from [Muse
+lab](https://www.tindie.com/products/johnnywu/pmod-vga-expansion-board/), rename
+file `colorBarVGA_muselab.ccf` to `colorBarVGA.ccf` (colors could be wrong, but
+syncro will be ok).
+
```

Comment:
Maybe it's more easy  to change
```
CONSTR = $(TOP).ccf
```
by
```
VGA ?= digilent
CONSTR=$(TOP)_$(VGA).ccf
```
and to rename `colorBarVGA.ccf` to `colorBarVGA_digilent.ccf`

so now with `make` or `make VGA=digilent` gateware will be build for the digilent PMOD and with `make VGA=muselab` gateware wiil be build for the muselab PMOD.

Reply:
Ok done ;)